apps/website/screens/components/chip/ChipPageLayout.tsx

```diff
@@ -17,9 +16,10 @@ const ChipPageHeading = ({ children }: { children: ReactNode }) => {
         <DxcFlex direction="column" gap="2rem">
           <ComponentHeading name="Chip" />
           <DxcParagraph>
-            Chips are elements that represent status, complementary information, or association between elements.
+            A <strong>chip</strong> is a compact, interactive UI element used to represent small pieces of information,
```

Comment:
I know in the docs they use strong, but I'm not sure if we should use it (in the rest of the components, there's no bold letters in the component definition).